pyproject.toml

```diff
@@ -18,13 +18,13 @@ dependencies = [
     "scikit-image", # secretly required by intake-xarray as default reader
     "torch",
     "xarray",
-    "resnet50-cefas@git+https://github.com/jmarshrossney/resnet50-cefas",
+    #"resnet50-cefas@git+https://github.com/jmarshrossney/resnet50-cefas",
 ]
 
 [project.optional-dependencies]
 jupyter = ["jupyterlab", "jupytext", "matplotlib"]
-dev = ["pytest", "black", "flake8", "isort"]
-all = ["cyto_ml[jupyter,dev]"]
+dev = ["pytest", "black", "flake8", "isort", "pip-tools", "conda-lock"]
+#all = ["cyto_ml[jupyter,dev]"]
```

Comment:
This pattern for collecting dependencies together also doesn't work with `conda-lock` as it tries to look up `cyto_ml` and doesn't find it.